src/Fusion/HTTP.elm

```diff
@@ -90,19 +91,13 @@ toHttpBody body =
             Http.emptyBody
 
         Fusion.Types.StringBody mime string ->
-            Http.stringBody mime string
+            Http.stringBody "application/json" string
```

Comment:
Is this a layover or intentional hardcoding? I figured the `StringBody` should be the "types we don't explicitly support" fallback.

Reply:
Oh yes, good catch! I'll fix that 👍